internal/metric/disk.go

```diff
@@ -38,6 +42,17 @@ func CollectDiskMetrics() (MetricsSlice, []CustomErr) {
 			continue
 		}
 
+		diskIOCounts, diskIOErr := disk.IOCounters(p.Device)
+		if diskIOErr != nil {
+			diskErrors = append(diskErrors, CustomErr{
+				Metric: []string{"disk.read_bytes", "disk.write_bytes", "disk.read_time", "disk.write_time"},
+				Error:  diskIOErr.Error() + " " + p.Device,
+			})
+			continue
+		}
+
+		stats, _ := diskIOCounts[p.Device]
```

Comment:
diskIOCounts stores data without `/dev/` prefix, we should remove the prefix to get real data.

```json
{
    "read_bytes": 11502867456,
    "write_bytes": 13696868352,
    "read_time": 82797,
    "write_time": 591770,
}
```

```suggestion
		deviceName := strings.TrimPrefix(p.Device, "/dev/")
		stats := diskIOCounts[deviceName]
```